### PR TITLE
add CenterPerCell transform and the PFlogPF composition

### PR DESCRIPTION
**Pull request by sjfleming**
Based on the results here

https://www.biorxiv.org/content/10.1101/2022.05.06.490859v3

This transform simply applies a per-cell centering to log1p(fractional_counts) data.

### CODE REVIEW

File: cellarium/ml/transforms/center_per_cell.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+class CenterPerCell(torch.nn.Module):
+    """
+    Center each cell by subtracting its mean across genes.
+
+    NOTE: PFlog1pPF / shifted-CLR-style normalization [1] is:
+        NormalizeTotal + Log1p + CenterPerCell
+
+    References:
+    [1] Booeshaghi, Hallgrimsdottir, Galvez-Merchan, Pachter. Depth normalization for single-cell genomics count data.
+        bioRxiv (2026). https://www.biorxiv.org/content/10.1101/2022.05.06.490859v3
```
Comment (fedorgrab) [inline cellarium/ml/transforms/center_per_cell.py]: Do you think we need to include this ref for CenterPerCell? I was thinking centering is a pretty standard operation, not really tight to this paper.

File: cellarium/ml/transforms/pflogpf.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+    This is a convenience wrapper for that sequence of transforms, but it provides
+    no additional functionality.
+
+    References:
+    [1] Booeshaghi, Hallgrimsdottir, Galvez-Merchan, Pachter. Depth normalization for single-cell genomics count data.
+        bioRxiv (2026). https://www.biorxiv.org/content/10.1101/2022.05.06.490859v3
+    """
+
+    def __init__(self, target_count: int = 1):
```
Comment (fedorgrab) [inline cellarium/ml/transforms/pflogpf.py]: I would make `target_count`'s default to be `10_000` to be consistent with the default value of NormalizeTotal, or is this this for a reason set to 1?

Comment (fedorgrab) [reply cellarium/ml/transforms/pflogpf.py]: Maybe we also need to expose eps for NormalizeTotal, even tho it's unlikely anyone would need to set a different value, but to have a better modular API, I would expose eps and put the same default value to it as in normalize total.

**Review by fedorgrab — APPROVED** [review]
I added couple of comments, I would consider them "nit"s. Looks very good, excited to see it in action!

File: cellarium/ml/transforms/center_per_cell.py (hunk 1 above)
Comment (sjfleming) [reply cellarium/ml/transforms/center_per_cell.py]: you might be right, yeah it's not something they invented... let's leave the reference in PFlogPF only

File: cellarium/ml/transforms/pflogpf.py (hunk 2 above)
Comment (sjfleming) [reply cellarium/ml/transforms/pflogpf.py]: Actually I had `target_count` as 10k at first, but I realized their formula in PFlogPF uses `target_count` 1.  It's just straight up normalization, not scaling to a new count value.

Okay yeah I'll expose `eps`

Comment (fedorgrab) [reply cellarium/ml/transforms/pflogpf.py]: Oh got it, okay then! I just thought was unintentional, if that's how they do it in paper, then makes sense!
